X-AIP.md

```diff
@@ -9,18 +9,14 @@ updated (*optional): <date created on, in ISO 8601 (yyyy-mm-dd) format> or N/A
 
 This is the template for AIPs.
 
-Note that the filename should contain the generated identifier by the author. When opening a pull request to submit your AIP, please use an abbreviated title in the filename, `identifier-AIP-draft_title_abbrev.md`.
+When opening a pull request to submit your AIP, please use an uppercase abbreviated title in the filename, `TITLE_ABBREV.md`.
 
 The title should be 44 characters or less.
 
 ## Simple Summary
 
 If you can't explain it simply, you don't understand it well enough. Provide a simplified and layman-accessible explanation of the AIP.
 
-## Abstract
```

Comment:
noone uses this and the few proposals i found using it, reward what's written in `#simple summary`